homeassistant/components/delijn/sensor.py

```diff
@@ -60,72 +64,48 @@ async def async_setup_platform(hass, config, async_add_entities, discovery_info=
 class DeLijnPublicTransportSensor(SensorEntity):
     """Representation of a Ruter sensor."""
 
-    _attr_device_class = DEVICE_CLASS_TIMESTAMP
+    _attr_device_class = SensorDeviceClass.TIMESTAMP
+    _attr_icon = "mdi:bus"
 
     def __init__(self, line):
         """Initialize the sensor."""
         self.line = line
-        self._attributes = {ATTR_ATTRIBUTION: ATTRIBUTION}
-        self._name = None
-        self._state = None
-        self._available = True
+        self._attr_attribution = ATTRIBUTION
+        self._attr_extra_state_attributes = {}
```

Comment:
@frenck is this needed?
Or is it better to create it if None inside async_update?

Reply:
🤷

Reply:
I think this is good to keep current behaviour.
Initialise empty in `__init__` and update it in `async_update`